Units/parser-python.r/type-statements.d/expected.tags

```diff
@@ -0,0 +1,16 @@
+MyList0	input.py	/^MyList0 = list[str]$/;"	v
+MyList1	input.py	/^MyList1: TypeAlias = list[str]$/;"	v	typeref:typename:TypeAlias
+MyList2	input.py	/^type MyList2 = list[str]$/;"	v	typeref:typename:TypeAlias
```

Comment:
Technically the `type` statement creates variables of type [`typing.TypeAliasType`](https://docs.python.org/3/library/typing.html#typing.TypeAliasType), which is a different thing from [`typing.TypeAlias`](https://docs.python.org/3/library/typing.html#typing.TypeAlias) (which is deprecated).  Thus it would be misleading to claim they're of type `TypeAlias`.

```suggestion
MyList2	input.py	/^type MyList2 = list[str]$/;"	v	typeref:typename:TypeAliasType
```